src/index.js

```diff
@@ -10,3 +14,15 @@ ReactDOM.render(
   document.getElementById('root'),
 );
 registerServiceWorker();
+
```

Comment:
The only time you're ever going to import store in your code should be in index.js and only to provide it in the provider component.